File: GeoQuizJava/app/src/main/java/com/akimov/android/geoquiz/Question.java
package com.akimov.android.geoquiz;

/**
 * Created by Alexander on 4/20/2016.
 */
public class Question {
    private int mTextResId;
    private boolean mAnswerTrue;
    private boolean mCheated;

    public Question(int textResId, boolean answerTrue) {
        mTextResId = textResId;
        mAnswerTrue = answerTrue;
    }
    public boolean isCheated() {
        return mCheated;
    }

    public void setCheated(boolean cheated) {
        mCheated = cheated;
    }
    public boolean isAnswerTrue() {
        return mAnswerTrue;
    }

    public void setAnswerTrue(boolean answerTrue) {
        mAnswerTrue = answerTrue;
    }

    public int getTextResId() {
        return mTextResId;
    }

    public void setTextResId(int textResId) {
        mTextResId = textResId;
    }
}

File: GeoQuiz/app/src/main/kotlin/com/akimov/geoquiz/domain/models/Question.kt
package com.akimov.geoquiz.domain.models

/**
 * Created by lex on 1/5/18.
 */
open class Question(var id: Int,
                    var questionText: String,
                    var isAnswerTrue: Boolean,
                    var description: String,
                    var isAnswerShown: Boolean = false)